lib/AST/Decl.cpp

```diff
@@ -5424,7 +5424,19 @@ void VarDecl::setSpecifier(Specifier specifier) {
                           StorageIsMutable_t(!isImmutableSpecifier(specifier)));
 }
 
-bool VarDecl::isAnonClosureParam() const {
+bool ParamDecl::isAnonClosureParam() const {
```

Comment:
Can you please extract this code into a separate static method something like `bool nameStartsWithDollarSign(DeclName)` and use to from both places?